src/main/java/uk/gov/pay/ledger/event/model/SalientEventType.java

```diff
@@ -5,7 +5,32 @@
 
 public enum SalientEventType {
```

Comment:
I wonder if we should rename this to `EventType` - feel like the salient concept has died

Reply:
@heathd suggested just using Strings in this map as the contract with Connector events is based on their name and the object isn't used anywhere else, what do you think? 

Happy to rename to `EventType` in the interim